test/test_rospy/test/unit/test_rospy_core.py

```diff
@@ -209,14 +336,6 @@ def test_is_topic(self):
                 self.fail(f)
             except rospy.core.ParameterInvalid:
                 pass
-            
-    def test_configure_logging(self):
-        # can't test actual functionality
-        try:
-            rospy.core.configure_logging("/")
-            self.fail("configure_logging should not accept a the root namespace as the node_name param")
-        except: pass
-        rospy.core.configure_logging("/node/name")
```

Comment:
somehow it didn't find that bug : https://github.com/ros/ros_comm/pull/1144/files#diff-84739190beb4f06aecd26a816f126e29L350
and it s now tested in context there: https://github.com/ros/ros_comm/pull/1144/files#diff-719224e090245a8163761263deeb3355R94